contracts/Blast_SpokePool.sol

```diff
@@ -0,0 +1,118 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.0;
+import "@eth-optimism/contracts/libraries/constants/Lib_PredeployAddresses.sol";
+
+import "./Ovm_SpokePool.sol";
+import "./external/interfaces/CCTPInterfaces.sol";
+
+// USDB and WETH on Blast accrue yield that can be claimed by any account holding the token. So for the length of
+// time that the SpokePool holds on to these assets, it can can claim interest.
+interface IERC20Rebasing {
+    function claim(address recipient, uint256 amount) external returns (uint256);
+
+    function getClaimableAmount(address account) external view returns (uint256);
+}
+
+/**
+ * @notice Blast Spoke pool.
+ */
+contract Blast_SpokePool is Ovm_SpokePool {
+    // This is the yield-accruing stablecoin on Blast that USDC/DAI/USDT all bridge into. It can be withdrawn
+    // from L2 into DAI.
+    address private constant USDB = 0x4300000000000000000000000000000000000003;
+    // Token that is received when withdrawing USDB, aka DAI.
+    address private constant L1_USDB = 0x6B175474E89094C44Da98b954EedeAC495271d0F;
```

Comment:
Same comment here: since this would likely be a different address on testnet, should we make it immutable and pass it in in the constructor?